src/airflow/airqo_etl_utils/message_broker.py

```diff
@@ -38,7 +38,7 @@ def send_data(
                 # partition = int(self.__partitions[partition_index])
                 range_data = data[i : i + 50]
 
-                message = {"data": range_data, "action": action}
+                message = {"data": range_data, "action": action, "tenant": tenant}
```

Comment:
Added tenant cc @Baalmart

Reply:
Thanks, so the package or library you are using allows the usage of custom fields? @NoahNsimbe

Reply:
I am not using a package or library to create the message. Everything is developed from scratch. I however use a package to connect to the message broker